packages/server/src/index.ts

```diff
@@ -210,9 +224,14 @@ export class App {
         })
 
         // Returns specific component node icon via name
-        this.app.get('/api/v1/node-icon/:name', (req: Request, res: Response) => {
-            if (Object.prototype.hasOwnProperty.call(this.nodesPool.componentNodes, req.params.name)) {
-                const nodeInstance = this.nodesPool.componentNodes[req.params.name]
+        this.app.get('/api/v1/node-icon/:name', param('name').notEmpty().escape(), (req: Request, res: Response) => {
+            const name = req.params.name
```

Comment:
for every validation, do we have to create another `const name`? can we just use the `req.params.name` so that when we remove this `notEmpty().escape()` check in future we have less work to do

Reply:
I have changed this and removed validation at each function level.